fender.bnf

```diff
@@ -69,7 +69,7 @@ typeAnnotation ::= ":" lineSep? name
 codeBody ::= "{" lineSep? (statement break)* statement? lineSep? "}"
 optionalArgs ::= "[" delimitedList<arg, ",", sep> vararg? "]"
 vararg ::= "..."
-args ::= "(" sep? delimitedList<arg, ",", sep> sep? ("," sep? optionalArgs)? sep? ")"
+args ::= ("(" sep? delimitedList<arg, ",", sep> sep? ("," sep? optionalArgs)? sep? ")") | "(" sep? optionalArgs sep? ")"
```

Comment:
Why not just make the delimited list of arguments preceding optionalArgs optional?